PAWSUITests/DataManagementTests.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  DataManagementTests.swift
+//  PAWSUITests
+//
+//  Created by Matthew Turk on 5/15/24.
+//
+
+import XCTest
+
+final class DataManagementTests: XCTestCase {
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        continueAfterFailure = false
+        let app = XCUIApplication()
+        app.launchArguments = ["--showOnboarding", "--useFirebaseEmulator"]
+        app.deleteAndLaunch(withSpringboardAppName: "PAWS")
+    }
+    
+    func testPullToRefresh() throws {
+        let app = XCUIApplication()
+        try app.navigateOnboardingFlow(email: "johndoe@stanford.edu")
+        let initialECGCount = app.staticTexts["ECG Recording"].exists
+        
+        try self.exitAppAndOpenHealth(.electrocardiograms)
+        app.activate()
+        
+        // Simulate pull to refresh.
```

Comment:
@MatthewTurk247 thank you for starting to work on the tests!

If everything goes as planned, the ECG should already be here once you get back into the app.

I would suggest to do the assertion check here, then do a pull-to-refresh, and then see that the same ECG is still here (meaning that the pull-to-refresh also loaded the on ECG that should be displayed when it replaced all the ECGs with the new query result)

Reply:
How does it look now, @PSchmiedmayer? UI tests seem to be working like a charm now, locally, but something seems to keep failing when running on the CI: https://github.com/StanfordBDHG/PediatricAppleWatchStudy/actions/runs/9122494282/job/25083360221?pr=61

Reply:
I watched the video recording of the UI test, and it looks like something is getting stuck on the invitation codes screen, even though that had not been an problem in recent commits. Do you know of anything in [1c3ac2e](https://github.com/StanfordBDHG/PediatricAppleWatchStudy/pull/61/commits/1c3ac2ec8acc29c37a1bd417932982ac5dc9bb25) could have had an effect?

Reply:
I suspect that this is due to the fact that the onboarding flow is run twice, meaning that the invitation code has already been used. It might make sense to add a flag to the `navigateOnboardingFlow` function that indicates if a user should sign up or log in. If they log in, you can skip the invitation code flow?

Reply:
Yup, it looks like that was the issue. I have now added a new invitation code to the persistent Firebase instance, so separate tests can use separate invitation codes when stepping through the onboarding flow.